tests/test_optimization.py

```diff
@@ -17,50 +17,94 @@
     "logisticregression__C": np.array([0.01,0.1,1,10,100,1000])
 }
 
+param_grid_empty = {}
+
+param_grid_invalid = {
```

Comment:
Doesn't look like this is used in a test - this would be a good test to add though!